jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/utils/PortalTests.java

```diff
@@ -1,30 +1,43 @@
 package io.github.epam.material.tests.utils;
 
 import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
-import static io.github.com.StaticSite.defaultPortalPage;
-import static io.github.com.StaticSite.portalFrame;
-import static org.hamcrest.Matchers.emptyString;
-import static org.hamcrest.Matchers.hasToString;
+import static io.github.com.MaterialNavigator.openSection;
+import static io.github.com.pages.utils.PortalPage.*;
+
+/**
+ * To see an example of Portal web element please visit
+ * https://material-ui.com/components/portal/.
+ */
+
 
 public class PortalTests extends TestsInit {
+
+    @BeforeMethod
+    public void openSectionToTest() {
+        openSection("Portal");
+        insureInitialState();
+    }
+
     @Test
-    public void portalTest(){
-        defaultPortalPage.open();
-
-        portalFrame.button.is().text("Mount children");
-        portalFrame.field1.is().text(hasToString("It looks like I will render here."));
-        portalFrame.field2.is().text(emptyString());
-
-        portalFrame.button.click();
-        portalFrame.button.is().text("Unmount children");
-        portalFrame.field1.is().text(hasToString("It looks like I will render here."));
-        portalFrame.field2.is().text(hasToString("But I actually render here!"));
-
-        portalFrame.button.click();
-        portalFrame.button.is().text("Mount children");
-        portalFrame.field1.is().text(hasToString("It looks like I will render here."));
-        portalFrame.field2.is().text(emptyString());
+    public void checkStateAfterBtnClick() {
+        button.click();
+        button.has().text("Unmount children");
+        field1.has().text("It looks like I will render here.");
+        field2.has().text("But I actually render here!");
+    }
```

Comment:
You don't use this method. Please use it, possible taking full code to test case